### PR TITLE
Add ImagePicker on register/update profile && change field in HelpView

### PR DESCRIPTION
**Pull request by idanlevi1**
I'm using a Cloudinary service to save images (with ImagePicker of Expo) on _registerView_
Change field in _HelpView_ because email isn't required field.

### CODE REVIEW

File: src/views/register/RegisterView.js [hunk 1]
```diff
@@ -49,16 +52,74 @@ class RegisterView extends React.Component {
...
+            console.log("avatarUrl:", pickerResult.uri)
+            this.setState({ avatarUrl: pickerResult.uri, disabled: false});
+            let base64Img = `data:image/jpg;base64,${pickerResult.base64}`
+            this.base64Img = base64Img;
+        }
+    };
+
+    updateAvatar = async () =>{
+        this.setState({spinner: true})
+        try {
```
Comment (sharvit) [inline src/views/register/RegisterView.js]: There is too much logic here.
React components shouldn't create the ajax request, they should use an external service for it. exactly what `pickImage` doing great :+1:

File: src/views/help/Help.js [hunk 2]
```diff
@@ -10,17 +10,18 @@ class Help extends Component{
...
             last={this.props.last} 
             email={this.props.email}
+            phone={this.props.phone}
             navigation={this.props.navigation}
         />
         )}
 }
 
-helpReqHandler = async (first,last,email,content) => {
+helpReqHandler = async (first,last,contact,content) => {
```
Comment (sharvit) [inline src/views/help/Help.js]: I would suggest moving this logic into `HelpService`.

File: src/views/register/RegisterView.js [hunk 3]
```diff
@@ -49,16 +52,74 @@ class RegisterView extends React.Component {
...
             behavior="padding">
             <View style={styles.userView}>
-                <TouchableOpacity>
-                <Image style = {styles.userImage} source ={require('../../images/emptyUserIcon.png')} />
+                <TouchableOpacity onPress={this.pickImage}>
+                    { this.state.avatarUrl ? 
+                        <Image style = {styles.userImage} source={{ uri: this.state.avatarUrl }} />
+                        :
+                        <Image style = {styles.emptyUserImage} source = {require('../../images/emptyUserIcon.png')} />  
+                    }
```
Comment (sharvit) [inline src/views/register/RegisterView.js]: Sometimes it makes it easier to understand when you do it as well:
```js
const avatarImage = this.state.avatarUrl ? (
  <Image style={styles.userImage} source={{ uri: this.state.avatarUrl }} />
) : (
  <Image
    style={styles.emptyUserImage}
    source={require('../../images/emptyUserIcon.png')}
  />
);

return (
  <TouchableOpacity onPress={this.pickImage}>{avatarImage}</TouchableOpacity>
)
```

File: src/views/register/RegisterView.js (hunk 1 above)
Comment (sharvit) [reply src/views/register/RegisterView.js]: I don't like the mixed usage of `async/await` with `then`.
I think the code will look much better if you replace all your `then` with `await`

File: src/views/register/RegisterView.js (hunk 3 above)
Comment (idanlevi1) [reply src/views/register/RegisterView.js]: Thanks for the review, I hope the changes are good now.
Nice to meet you

**Pull request merged**